executor/diagnostics.go

```diff
@@ -138,10 +138,10 @@ func (configInspection) inspect(_ context.Context, sctx sessionctx.Context, filt
 			continue
 		}
 		results = append(results, inspectionResult{
-			item:      row.GetString(1),                   // key
-			value:     fmt.Sprintf("%d", row.GetInt64(2)), // count
-			reference: "1",
-			severity:  "P2",
+			item:      row.GetString(1), // key
+			value:     "inconsistent",
+			reference: "consistent",
```

Comment:
Since you changed numbers to strings, change the fields to 'actual' and 'expected' may be better.

Reply:
I think it's better to change the `reference` to `expected` but keep the `value` unchanged. The `value` may be applied to more rules properly.